app/policies/wiki_policy.rb

```diff
@@ -0,0 +1,12 @@
+class WikiPolicy
+  attr_reader :user, :wiki
+
+  def initalize(user, wiki)
```

Comment:
The method name here is misspelled, it should be `initialize`. The error message in Pundit is a bit confusing in this case, I filed an issue for it [here](https://github.com/elabs/pundit/issues/448).